data/suspend/index.js

```diff
@@ -35,11 +38,15 @@ document.querySelector('h2').textContent = search.url || '...';
     ctx.arc(img.width * 0.75, img.height * 0.75, img.width * 0.25, 0, 2 * Math.PI, false);
     ctx.fill();
 
-    document.querySelector('link[rel*="icon"]').href = canvas.toDataURL('image/ico');
+    setFavicon(canvas.toDataURL('image/ico'));
   };
-  img.src = 'chrome://favicon/' + search.url;
+  img.src = search.favicon || chrome.extension.getURL('img/default.ico');
```

Comment:
In Chrome and Opera `'chrome://favicon/' + search.url` one works just fine. Only use `search.favicon` for Firefox version